src/main/java/com/commercetools/pspadapter/paymentHandler/impl/InterfaceInteractionType.java

```diff
@@ -0,0 +1,20 @@
+package com.commercetools.pspadapter.paymentHandler.impl;
+
+public enum InterfaceInteractionType {
+    REQUEST("paypal-plus-interaction-request"),
+    RESPONSE("paypal-plus-interaction-response");
+
+    private final String interfaceKey;
+
+    InterfaceInteractionType(String interfaceKey) {
+        this.interfaceKey = interfaceKey;
+    }
+
+    public String getInterfaceKey() {
+        return interfaceKey;
+    }
+
+    public String getValueFieldName() {
```

Comment:
The enum name should not be lowercase as it's a convention. On the other hand, I want to have a lowercase name for a field "value" in the interface interaction in CTP.

Reply:
then extend the enum to have 2 string constants:
```java
REQUEST("request", "paypal-plus-interaction-request"),
RESPONSE("response", "paypal-plus-interaction-response");

private final String fieldName;
private final String interfaceKey;
```

Otherwise it is malformed enum usage: the enum value name should be only programmer specific, we should not relay on the constant names on the custom field values.

Reply:
https://github.com/commercetools/commercetools-paypal-plus-integration/pull/49/commits/e9673dc9408a46f2536ef9a5dca2648b34a3d7d0